pages/workshop/Skew_T/SkewT_and_Hodograph.ipynb

```diff
@@ -465,7 +461,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "skew.ax.plot(lcl_temperature, lcl_pressure, marker=\"_\", color='k', markersize=30, markeredgewidth=3)\n",
```

Comment:
Not a fan of using the horizontal line marker rather than a full horizontal line? This was an intentional change before, so I'm curious on your thoughts.

Reply:
Well now I'm confused because later ones are using the markers...

Reply:
I thought the horizontal line marker had a bit more explaining/syntax so I removed it from the first time we mark the LCL, but then left it in later so we could show "another" way to do it. Seem fair?

Reply:
Sounds reasonable.